geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/wancommand/StartGatewaySenderCommandDUnitTest.java

```diff
@@ -335,6 +335,103 @@ public void testStartGatewaySender_Group_MissingSenderFromGroup() throws Excepti
     server3.invoke(() -> verifySenderState("ln", true, false));
   }
 
+  @Test
+  public void testStartGatewaySender_clean_queues() throws Exception {
```

Comment:
I don't understand what these two tests are testing. I see that one cleans the queue and the other doesn't, but they have the same assertions on the expected behavior. Shouldn't the assertions be different since they do different things?

Reply:
Thist test is to verify, that new option "clean-queues" is supported, and to verify option values